browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3880,6 +3880,21 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_CONTINUOUS_LISTENING_SUMMARY" desc="Continuous Listening summary for playlist settings">
       Automatically plays the next items in your playlist
     </message>
+    <message name="IDS_BRAVE_WALLET_ACTIVITY" desc="Wallet activity and transactions">
+      Activity
+    </message>
+    <message name="IDS_DATE_TEXT" desc="Date text as heading">
+      Date
+    </message>
+    <message name="IDS_MARKET_PRICE_TEXT" desc="Market price text title as heading of an asset">
+      Market Price
+    </message>
+    <message name="IDS_BRAVE_WALLET_NO_TRANSACTIONS_YET" desc="Text for when zero transactions are currently availble for viewing">
+      No transactions yet
+    </message>
+    <message name="IDS_BRAVE_WALLET_NO_TRANSACTIONS_YET_DESCRIPTION" desc="Explaination of why the transaction list is empty, and what to expect after the first transaction is available">
+      After your first transaction you will be able to view it here.
```

Comment:
This string could be improved, cc: @rmcfadden3 could you please assist.

Reply:
Thanks, updated the desktop copies as well: [b46a4eb](https://github.com/brave/brave-core/pull/18631/commits/b46a4eb20507245f4bfbdeedf0ce356f651b5cdc)

Reply:
Sounds good to me. Thank you @rmcfadden3 and @Pavneet-Sing